grammar.js

```diff
@@ -1793,6 +1796,21 @@ module.exports = grammar({
       ),
     ),
 
+    between_expression: $ => choice(
+      ...[
+            [$.keyword_between, 'between'],
+            [seq($.keyword_not, $.keyword_between), 'not_between'],
+        ].map(([operator, precedence]) =>
+                prec.left(precedence, seq(
+                field('left', $.identifier),
```

Comment:
Another option, is to use field here and then try to detangle the precedences, but I couldn't get that to work:

```suggestion
                field('left', $.field),
```